endpoints/telegram.py

```
@@ -12,28 +13,47 @@
+                receiver = str(args[1])
+                amount = int(args[2])
+                if amount <= 0:
+                    self.bot.send_message(str(message.from_user.id), "Error: You can`t send negative or zero amount of tokens")
+                    return
+                if not self.users_service.is_user_registered(receiver):
+                    self.bot.send_message(str(message.from_user.id), "Error: the receiver does not exist. Double check receiver\'s user_id")
+                    return
+                result = self.users_service.transact(str(message.chat.id), receiver, amount)
+                if not result:
+                    self.bot.send_message(str(message.from_user.id), "Error: insufficient balance")
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)